src/kokkos_ext/ArborX_DetailsKokkosExtStdAlgorithms.hpp

```diff
@@ -9,27 +9,17 @@
  * SPDX-License-Identifier: BSD-3-Clause                                    *
  ****************************************************************************/
 
-#ifndef ARBORX_DETAILS_KOKKOS_EXT_SWAP_HPP
-#define ARBORX_DETAILS_KOKKOS_EXT_SWAP_HPP
+#ifndef ARBORX_DETAILS_KOKKOS_EXT_STD_ALGORITHMS_HPP
+#define ARBORX_DETAILS_KOKKOS_EXT_STD_ALGORITHMS_HPP
 
-#include <Kokkos_Macros.hpp>
+#include <Kokkos_StdAlgorithms.hpp>
```

Comment:
This is not desirable

Reply:
Please elaborate.

Reply:
Pulling Kokkos.Core plus all of the Kokkos parallel algorithms to use a simple template function.